sdk/Lusid.Sdk.Tests/tutorials/Instruments/ComplexInstruments.cs

```diff
@@ -50,6 +50,8 @@ public void DemonstrateCreationOfSwap()
 
             // CREATE the flow conventions, index convention
             var flowConventions = new FlowConventions(
+                scope: null,
```

Comment:
why do you have to explicitly set this to null? it should default to null if not required. what is the default in other languages?